common/djangoapps/third_party_auth/tests/specs/test_google.py

```diff
@@ -53,7 +53,7 @@ def test_custom_form(self):
         """
         Use the Google provider to test the custom login/register form feature.
         """
-        # The pipeline starts by a user GETting /auth/login/google-oauth2/?auth_entry=custom1
+        # The pipeline starts by a user Getting /auth/login/google-oauth2/?auth_entry=custom1
```

Comment:
Eh - again, please leave it alone. `GET` is the HTTP method, so all-caps is correct. The only change I could see would be removing the extraneous T, but I like a clean diff better than I like perfectly-clean code comments.